Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM manjarolinux/base:latest
+
+ARG VERSION=1.29.0
+
+# This will run as root
+RUN pacman --noconfirm -Syu base-devel go ruby php-embed nodejs npm
```

Comment:
Question: does it matter which version of these languages are installed for the build?

Reply:
Not currently, will add versioning if this breaks in the future, meanwhile current packages should be used.